.github/workflows/robustness.yaml

```diff
@@ -9,3 +9,11 @@ jobs:
       count: 15
       testTimeout: 30m
       artifactName: main
+  main-arm64:
+    uses: ./.github/workflows/robustness-template.yaml
+    with:
+      etcdBranch: main
+      count: 15
+      testTimeout: 30m
+      artifactName: main-arm64
+      runs-on: "['self-hosted', 'Linux', 'ARM64']"
```

Comment:
Based on our previous discussion, we only want to nightly run for ARM64, to avoid run test on each PR, because we have only two ARM64 servers.

Reply:
Must have missed this in discussion somewhere, makes sense, updated to nightly only.